skyportal/handlers/api/gcn.py

```diff
@@ -914,7 +914,7 @@ def delete(self, dateobs):
 
 class GcnEventTagsHandler(BaseHandler):
     @auth_or_token
-    async def get(self, dateobs=None, tag=None):
+    async def get(self, *ignored_args):
```

Comment:
This changes those args from keyword to positional?

Reply:
so ew have a couple of endpoints where the GET, POST, DELETE take a different number of arguments (for example the GET, DELETE might take an id that the POST does not). To avoid duplicating the endpoint in the `app_server.py` the easiest way to fix this is to actually add the unused parameter to the handler, with a None value. However the issue with that is that these the documentation would still show that unused argument in the endpoint URL that it generates, as if it was required or smth.

Here all that I did is look for handlers with unused arguments, make those `*args` with a specific name (here, ignored_args) so that in `openapi.py` I ignore those specifically, and can end up with the correct & more concise endpoints).

Does that make sense? 

Oh and actually opening another small PR on baselayer to help display the permissions required to use an endpoint (if any), which should help remove a lot of the confusion our users have at the moment!

Reply:
Totally fine, was simply wondering whether you wanted `**ignored_args` instead?

Reply:
ah well that way I had a single argument name I can look for an ignore when autogenerating the openapi specs, instead of having to maintain a list of args. But I mean I could simply rename all of those to idk, something like `unused_arg` and avoid those in a loop, but *args seemed appropriate as it automatically throws all of the unused arguments in a list that I can ignore all at once.

Reply:
I'll ping you on Slack; I am really only wondering about the argument vs keyword distinction.